### DISCUSSION TITLE
这个项目感觉很不错

### DISCUSSION THREAD

**Discussion by huyanghun99** [shion-app/shion#3] [General]
以前一直有用Manic Time做时间记录和分析管理，但是Manictime只有记录，看到这个项目还可以结合自己的计划和任务，感觉很不错。

**Comment by hanaTsuk1** [comment]
感谢使用和支持🎉🎉🎉

**Reply by huyanghun99** [reply]
我是一名产品经理，并且对GTD有一定研究和实践(https://mp.weixin.qq.com/s/KsMRoE0MjBNH4b661NBROA)
不知道Shion后续的规划和发展是什么。

如果在GTD方向的话，我感觉可以提供一些帮助。

**Reply by hanaTsuk1** [reply]
我需要说明一个关键的信息，该项目更关注**做过了什么**而不是**计划要做什么**
我认为已经有太多的todo list类型的应用了
但还是非常感谢你想要助一份力

至于规划，我还有一部分功能计划实现，至于在完成这些之后需要做什么，我现在不能立刻给出回复
表演是需要演员和观众共同组成的，我需要收集更多的反馈，来决定下一步

不过，下一个版本**v0.10.0**的功能很明确，就是**搜索**

**Reply by huyanghun99** [reply]
嗯，很棒有清晰的定位和计划。

如果需要用户反馈的话，类似软件研究过ManicTime和RescueTime 他们都是付费软件，口碑蛮好的。他们有一些我觉得很好的特性，可以考虑在项目中是否要实现：

1.自动化标签，通过配置条件自动进行标签归类；例如打开XXX文档、XX站点，这段时间其实就是把时间放在XX项目上，这样可以把一个项目零碎的时间统计出来。或者XX网站，其实就是学习，这样就能把学习的时间统计出来；
2.RescueTime 中有一个归类(不知道怎么实现的)，把学习时间、项目时间、交流时间进行了统计；
3.outlook/钉钉等日程信息同步，毕竟什么时候开了一个什么会在工作中也是重要的时间项目
4.离开时间标记，当一段时间电脑处于非操作状态时，时间可能花在给客户打电话……，Manictime会弹出一个框，让用户标记这段时间干什么去了。这样可以一定程度补足自动采集的盲区，提高时间复盘完整性；
5.数据管理，目前看应该数据默认都缓存在C盘的？很多C盘不够用，强烈建议数据存储位置可以自定义

仅供参考，其他想到在反馈。对这个项目蛮感兴趣，我会持续关注和试用。

另外，稍微澄清一下，GTD不仅是做计划的定位，复盘做过什么是非常重要的环节。也是让人减轻工作压力和提高效率的理论和实践。
